packages/react/kibana_context/root/root_provider.tsx

```diff
@@ -17,7 +17,7 @@ export interface KibanaRootContextProviderProps extends KibanaEuiProviderProps {
   /** The `I18nStart` API from `CoreStart`. */
   i18n: I18nStart;
   /** The `AnalyticsServiceStart` API from `CoreStart`. */
-  analytics?: AnalyticsServiceStart;
+  analytics?: Pick<AnalyticsServiceStart, 'reportEvent'>;
```

Comment:
Using more specific interface here, to meet the shape of `DashboardAnalyticsService`